### ISSUE TITLE
Fix banner paragraph text colour (dark colour on dark) bug

### ISSUE THREAD

**Issue by mmafe** [ccswbs/hjckrrh#752]
![image](https://cloud.githubusercontent.com/assets/1927902/26456146/3af5d5c2-413a-11e7-8a1b-0157e21e4fdc.png)

The white paragraph banner text is getting overridden by the new colour. This relates to pull request #666 and issue #642.

**Comment by mmafe** [comment]
Recommend updating .slidesjs-slide-text so it has colour #FFFFFF.
You will also need to ensure any links inside of .slidesjs-slide-text have a colour of #428BCA (which still passes on black).

**Comment by mmafe** [comment]
FYI: currently this will not show up on production sites that have not yet cleared their cache. A release may be going out next Thursday (June 1st). If we want to be part of that release, the pull request would have to be done (ideally) by Monday.

In the meantime, if a site is affected, we can use CSS injector to fix temporarily.
